docs/source/index.rst

```diff
@@ -43,6 +43,8 @@ simulating decennial censuses, surveys, taxes, and other administrative data).
 By creating realistic, but simulated, data which includes these attributes, we
 can make ER research and development easier for ourselves and others.
 
+Want to know more? Please visit `the pseudopeople website <https://www.pseudopeople.org/>`_, where you can find out more about the principles and processes underlying this work.
```

Comment:
```suggestion
Want to know more? Please visit the `pseudopeople project website <https://www.pseudopeople.org/>`_, where you can find out more about the principles and processes underlying this work.
```
"The pseudopeople website" sounds ambiguous/confusing to me -- e.g., why isn't the pseudopeople documentation we're looking at here the same as the pseudopeople website? Os's website itself refers to "the pseudopeople project," so I think "the pseudopeople project website" seems like an appropriate name for it that helps distinguish it from the documentation, but I'm open to other suggestions.

Also, it seems like "the" doesn't need to be part of the link, but that's just a minor style thing.